### PR TITLE
DAT-16000      liquibase-redshift using Liquibase Parent POM

### PR DESCRIPTION
**Pull request by jandroav**
https://datical.atlassian.net/browse/DAT-16000

### CODE REVIEW

File: .github/dependabot copy.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: 2
```
Comment (filipelautert) [inline .github/dependabot copy.yml:1]: should this file be here?

**Review by filipelautert — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -185,27 +174,4 @@ jobs:
 #        uses: actions/upload-artifact@v3
 #        with:
 #          name: redshift-test-results
-#          path: build/spock-reports
-
-  dependabot:
```
Comment (filipelautert) [inline .github/workflows/test.yml:190]: This dependabot merge part... was it moved elsewhere?

**Review by filipelautert — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (jandroav) [reply .github/workflows/test.yml:190]: added back here https://github.com/liquibase/liquibase-cassandra/pull/218/commits/d5004627eb090b34cb6bce99fe4dfd20837cce17

Comment (filipelautert) [reply .github/workflows/test.yml:190]: one more thing :) It should need "unit-test" , not build - so it will only merge if tests are passing .

Comment (jandroav) [reply .github/workflows/test.yml:190]: done @filipelautert

**Pull request merged**